index.html

```diff
@@ -72,5 +75,49 @@
         ><a href="">Vaccine</a><a href="">Capital Riots</a>
       </div>
     </div>
+    <footer class="container footer1">
+      <div class="justify-content-between sub-footer">
+        <div class="sub-footer2">
+          <div class="footer-image">
+            <a href="">
+              <img
+                src="https://www.venrock.com/wp-content/uploads/2015/11/170361LOGO.jpeg"
+                alt=""
+              />
+            </a>
+          </div>
+          <p class="align-self-end paragraph1 justify-self-center">
+            <i class="far fa-copyright"></i> 2020 NEWSWEEK
+          </p>
+        </div>
+        <div
+          class="d-flex justify-content-between align-self-center icons mr-3"
+        >
+          <a href=""><i class="fab fa-facebook-square"></i></a>
+          <a href=""><i class="fab fa-twitter"></i></a>
+          <a href=""><i class="fab fa-instagram"></i></a>
+          <a href=""><i class="fab fa-tumblr"></i></a>
+          <a href=""><i class="fab fa-linkedin-in"></i></a>
+        </div>
+      </div>
+      <div class="footer2 mt-3 mb-1">
+        <span class="font-weight-bold"> Edition:</span> &nbsp;
+        <a href="#"> US Edition</a> |<a href="#"> 日本</a> |
+        <a href="#">Pakistan</a> |<a href=""> Polska </a>|<a href="">
+          Romania</a
+        >
+
+        <div class="footer-links pb-3 mb-1 mt-1">
+          <a href=""> About Us</a> | <a href="">Corrections</a> |
+          <a href="">Contact Us</a> | <a href="">Editorial Guidelines</a> |
+          <a href="">Advertise</a> | <a href="">Copyright</a> |<a href="">
+            Terms & Conditions</a
+          >
+          | <a href="">Privacy Policy</a> | <a href=""> Cookies Policies</a> |
+          <a href=""> Terms of Sales</a> |<a href=""> Archives</a> |
+          <a href=""> Announcement </a> | <a href="">Consent Preferences</a>
+        </div>
+      </div>
+    </footer>
```

Comment:
- [ ] Your footer should stick to the bottom of the page even if there is no content. You can make your footer stick to the bottom by adding the following bootstrap class `fixed-bottom`